realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -2918,39 +2613,6 @@ public void isEmpty_illegalFieldTypeThrows() {
                     case MIXED:
                         realm.where(AllJavaTypes.class).isEmpty(AllJavaTypes.FIELD_MIXED).findAll();
                         break;
-                    case INTEGER_LIST:
```

Comment:
As previously pointed out (on skipping the magical indexes) this change seems like a semantic change and should be highlighted in the CHANGELOG